pypeit/core/coadd2d.py

```diff
@@ -496,7 +498,7 @@ def rebin2d(spec_bins, spat_bins, waveimg_stack, spatimg_stack, thismask_stack,
 
 # TODO Break up into separate methods?
 
-class Coadd2d(object):
+class CoAdd2d(object):
```

Comment:
Yes, what should we name them to avoid collisions? One option would be to move coadd1d and coadd2d to one coadd.py module in core, but that would be a lot of code.

Reply:
I'm okay with that.  I'm also okay if we wait on this for a separate PR.

Reply:
Okay, I've put all the core.coadd1d and core.coadd2d in one core.coadd1 file and the CoAdd1d and 2d classes are now in pypeit